jenkinsfiles/Jenkinsfile.deploy-builder

```diff
@@ -1,27 +1,43 @@
-def ansibleConfig = evaluate readTrusted('jenkinsfiles/library/createAnsibleCfg.groovy')
-def vmConfig = evaluate readTrusted('jenkinsfiles/library/createVMConfig/deploy-builder.groovy')
+def commonVars
+def ansibleConfig
+def vmConfig
 
 pipeline {
   agent none
   stages {
     stage('Prepare environment') {
       agent { label 'ansible' }
       environment {
-        VC = credentials('b5d73edd-96c2-467d-8b97-bf52c0ec946a')
+        VC = credentials('vcenter')
       }
       steps {
-        git branch: 'ansible-juni', url: 'git@github.com:codilime/juniper-windows-internal'
         script {
-          vmConfig.create(env.VC_USR, env.VC_PSW, env.VM_TEMPLATE)
-          ansibleConfig.create(env.ANSIBLE_VAULT_KEY_FILE)
+          commonVars = evaluate readFile('jenkinsfiles/library/createCommonVars.groovy')
+          ansibleConfig = evaluate readFile('jenkinsfiles/library/createAnsibleCfg.groovy')
+          vmConfig = evaluate readFile('jenkinsfiles/library/createVMConfig/deployBuilder.groovy')
+        }
+
+        dir('ansible') {
+          script {
+            commonVars.create(env.SHARED_DRIVE_IP, env.JENKINS_MASTER_IP)
+            ansibleConfig.create(env.ANSIBLE_VAULT_KEY_FILE)
+            vmConfig.create(env.VC_HOSTNAME, env.VC_DATACENTER, env.VC_CLUSTER, env.VC_FOLDER,
+                            env.VC_NETWORK, env.VC_USR, env.VC_PSW, env.VM_TEMPLATE)
+          }
+          sh 'cp inventory.sample inventory'
+          sh 'ansible-galaxy install -r requirements.yml -f'
```

Comment:
There is a lot of common code between this diff and https://github.com/codilime/contrail-windows-ci/blob/master/jenkinsfiles/Jenkinsfile.create-builder-template#L9-L30

I think we should extract this to some library function, that takes `vmConfig` as parameter (since this job takes `deployBuilder.groovy` and the linked one takes `createBuilderTemplate.groovy` and it's the only differecne that I can see).